### PR TITLE
DragonflyBSD: instructions on testing efivar port

### PR DESCRIPTION
**Pull request by SergiiDmytruk**

### CODE REVIEW

File: dragonflybsd/libs.md [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+ * build a test tool
+ * verify that it works
+
+## Preparing for using ports
+
+`git`, `gcc` and other basic tools should be installed by default, so we only
+need to pull ports tree. Run this and other commands below as a `root` user:
+
+```
+git clone --depth 1 -b efivar https://github.com/3mdeb/DPorts.git /usr/ports
```
Comment (macpijan) [inline dragonflybsd/libs.md]: @SergiiDmytruk There is something wrong with the paths - either here or in the ports. Should it be `/usr/dports` here instead? If i change to `dports` (here and below) it works.

File: dragonflybsd/libs.md [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+
+`-rpath` is needed to avoid picking up `/usr/lib/libefivar.so` which comes with
+the system. Might need to rename port's library to get rid of this.
+
+## Use the test tool
+
+### Adding/updating a variable
+
+```
+./efivarlibtest set aaaaaaaa-bbbb-cccc-dddd-eeeeeeeeeeee test newval
```
Comment (macpijan) [inline dragonflybsd/libs.md:58]: @SergiiDmytruk I'm getting: `Failed to set var: Function not implemented`

File: dragonflybsd/libs.md [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+```
+
+### Reading a variable
+
+```
+./efivarlibtest get aaaaaaaa-bbbb-cccc-dddd-eeeeeeeeeeee test
```
Comment (macpijan) [inline dragonflybsd/libs.md:64]: @SergiiDmytruk As above

File: dragonflybsd/libs.md (hunk 1 above)
Comment (SergiiDmytruk) [reply dragonflybsd/libs.md]: Sure, I copied and edited OpenBSD's file and missed this.

File: dragonflybsd/libs.md (hunk 2 above)
Comment (SergiiDmytruk) [reply dragonflybsd/libs.md:58]: Added `kldload efirt` command. I always forget that `/dev/efi` is in a module here.

**Comment by macpijan** [comment]
@SergiiDmytruk Thanks it now works great for me.

**Pull request merged**